### DISCUSSION TITLE
Where is the settings save data located?

### DISCUSSION THREAD

**Discussion by CypherActive** [KadeArchive/Kade-Engine#775] [Q&A]
Hello, I would like to know where the save data is located for all the settings since my settings are glitched, thank you. Also just to know it does this for a few versions of kade engine, meaning old ones too so if the location has changed please tell me the locations.

![image](https://user-images.githubusercontent.com/85506969/121046840-ff370e80-c7ad-11eb-9f99-60fd562a8197.png)

**Comment by pepperly-art** [comment]
Assuming windows and depending on your version:

%appdata% -> Roaming/ninjamuffin99/Kade Engine/ninjamuffin99
1.5.0+ seems to be
%appdata% -> Roaming/KadeDev/Kade Engine

funkin.sol is the actual file

**Comment by isakube** [comment]
LMAO WHAT IS THAT OFFSET
what kinda bluetooth earbuds you using that require that /j
